Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,27 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
 # final image needs kubectl, so we copy `oc` from cli image to use as kubectl.
-FROM openshift/origin-cli:latest as cli
+FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
```

Comment:
As compared to `FROM openshift/ose-metering-helm:latest as helm` in `Dockerfile.rhel`, is hopefully identical image.

Reply:
are you just setting this for local dev/building?  during CI builds this value should be getting replaced (if it's not, we should look into why not)

Reply:
Are you referring to the URL host portion: `registry.svc.ci.openshift.org`? Either way, no this is as intended for CI builds, and currently it's working, so if it's working by taking advantage of a bug, let me know.

Reply:
this will "work" but it would probably be better to do the substitution in the release config so you don't have to change the tag here every time you branch.

example:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-master.yaml#L50

ensures this gets substituted:
https://github.com/openshift/origin/blob/master/images/hyperkube/Dockerfile.rhel#L1

with the image pointed to by the "bin" tag within the CI process.

(well it should, actually it's broken since the substitution is looking for 1.12 and the dockerfile contains 1.13)

Reply:
We're already substituting these references to use the corresponding ocp imagestream: https://github.com/openshift/release/blob/master/ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml#L2-L9, so this would be nice for local dev purposes.

Reply:
yeah for local dev, not quite sure the one, but not the other: 
```
Step 2/26 : FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
 ---> 072597ea8205
Step 3/26 : FROM registry.svc.ci.openshift.org/ocp/4.4:base as base-4-4
Get https://registry.svc.ci.openshift.org/v2/ocp/4.4/manifests/base: unauthorized: authentication required
```